vscode-lean4/src/leanclient.ts

```diff
@@ -337,7 +339,7 @@ export class LeanClient implements Disposable {
             diag.fullRange = p2c.asRange(protDiag.fullRange)
             return diag
         }
-        p2c.asDiagnostics = (diags) => diags.map(d => p2c.asDiagnostic(d))
+        p2c.asDiagnostics = async (diags) => diags.map(d => p2c.asDiagnostic(d))
```

Comment:
this is probably due to the vscode version change, please revert this since Gabriel wants this to be done in a separate PR.

Reply:
You can find this on: #162